zulipterminal/config/ui_mappings.py

```diff
@@ -73,3 +73,19 @@
     4: "Only organization administrators and moderators can edit topic",
     5: "Any user can edit topic",
 }
+
+SPECIAL_KEYS_MAPPINGS = {
+    "meta": "Meta",
+    "esc": "Esc",
+    "ctrl": "Ctrl",
+    "page up": "PgUp",
+    "page down": "PgDn",
```

Comment:
We don't need to do the page up/down part straight away, but it's good to see that it works, so we can see how it goes :+1: